packages/one-app-dev-bundler/esbuild/plugins/font-loader.js

```diff
@@ -19,7 +19,7 @@ import fs from 'fs';
 const fontLoader = {
   name: 'fontLoader',
   setup(build) {
-    build.onLoad({ filter: /.ttf$|.woff$|.woff2$|.jfproj$/ }, async (args) => {
+    build.onLoad({ filter: /\.(ttf|woff2?|jfproj)$/i }, async (args) => {
```

Comment:
I left it out of the commit message 😓 but one benefit to grouping the or is we only need to escape the period once